src/pswiggle.c

```diff
@@ -220,45 +220,49 @@ static int usage (struct GMTAPI_CTRL *API, int level) {
-	GMT_Message (API, GMT_TIME_NONE, "\t   Append +w<length> to set the scale length in data z-units.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Use +a to move label to the opposite side of vertical scale bar.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Use +l to set the unit label of the z-values for the scale bar label [no label].\n");
+	GMT_Usage (API, 1, "\n-C<center>");
+	GMT_Usage (API, -2, "Set center value to be removed from z before plotting [0].");
+	gmt_refpoint_syntax (API->GMT, "\n-D", "Specify position and dimensions of the vertical scale bar.", GMT_ANCHOR_VSCALE, 3);
+	GMT_Usage (API, 3, "+w Append <length> to set the scale length in data z-units.");
+	GMT_Usage (API, 3, "+a Move label to the opposite side of vertical scale bar.");
+	GMT_Usage (API, 3, "+l Set the unit label of the z-values for the scale bar label [no label].");
 	gmt_mappanel_syntax (API->GMT, 'F', "Specify a rectangular panel behind the vertical scale.", 4);
 	gmt_fill_syntax (API->GMT, 'G', NULL, "Specify color/pattern for positive and/or negative areas.");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Append +p to fill positive areas only (Default).\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Append +n to fill negative areas only.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Append both to fill positive and negative areas.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-I Set fixed projection azimuths for wiggles.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Azimuths of the normals to the track are reset to <az>.\n");
-	GMT_Option (API, "K");
-	GMT_Message (API, GMT_TIME_NONE, "\t-N Fill negative wiggles instead [Default is positive].\n");
+	GMT_Usage (API, -2, "You can control which areas are to be painted:");
+	GMT_Usage (API, 3, "+p Fill positive areas only (Default).");
+	GMT_Usage (API, 3, "+n Fill negative areas only.");
+	GMT_Usage (API, -2, "Append both modifiers to fill positive and negative areas.");
+	GMT_Usage (API, 1, "\n-I<az>");
+	GMT_Usage (API, -2, "Set fixed projection azimuths for wiggles. "
+		"Azimuths of the perpendiculars to the track are reset to <az>.");
+	GMT_Option (API, "Z,K");
 	GMT_Option (API, "O,P");
-	GMT_Message (API, GMT_TIME_NONE, "\t-T Specify track pen attributes. [Default is no track].\n");
+	GMT_Usage (API, 1, "\n-T<pen>");
+	GMT_Usage (API, -2, "Specify track pen attributes [Default draws no track].");
 	GMT_Option (API, "U,V");
-	gmt_pen_syntax (API->GMT, 'W', NULL, "Specify outline pen attributes [Default is no outline].", 0);
+	gmt_pen_syntax (API->GMT, 'W', NULL, "Specify wiggle outline pen attributes [Default is no outline].", 0);
 	GMT_Option (API, "X");
-	GMT_Message (API, GMT_TIME_NONE, "\t-Z Give the wiggle scale in data-units per %s.\n",
-		API->GMT->session.unit_name[API->GMT->current.setting.proj_length_unit]);
-	GMT_Message (API, GMT_TIME_NONE, "\t   Alternatively, append any unit from among %s [c].\n", GMT_DIM_UNITS_DISPLAY);
 	GMT_Option (API, "bi3,c,di,e,f,g,h,i,p,qi,t,w,:,.");
```

Comment:
```suggestion
	GMT_Option (API, "bi3,c,di,e,f,g,h,i,p,qi,s,t,w,:,.");
```

Reply:
Actually, -s is for output which is not what wiggle does.  I am removing it from the usage and leaving this as is.